### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by KevCaz**
Add content for Part 1 of the workshop!

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -72,37 +75,69 @@ class: inverse, center, middle
...
+
+### Understand how `r gt()` works
+
+--
+
+### Be able to use `r gt()`  and `r gh()` to collaborate
+
+--
+
+### Use [<img src="img/rmarkdown.png" style="vertical-align: middle;margin:10px" width="50px"></img>](https://rmarkdown.rstudio.com/) to create documents and expose them on `r gh()`
```
Comment (david-beauchesne) [inline index.Rmd:136]: I added an objective here for rmarkdown, if it makes sense to you just accept it

File: _01_git_github.Rmd [hunk 2]
```diff
@@ -27,135 +62,413 @@ class: inverse, center, middle
...
+
+
+
+---
+# `r gt()` servers
+
+
+## Your own server
+
+- e.g. a minimal ssh server (see this post)
```
Comment (david-beauchesne) [inline _01_git_github.Rmd:339]: What is the post? Is it the one about GitLab, or is the link missing?

**Review by david-beauchesne — APPROVED** [review]
All good on my side, good work as usual. I only did minor modifications and comments. Good job!

File: _01_git_github.Rmd (hunk 2 above)
Comment (KevCaz) [reply _01_git_github.Rmd:339]: A blog that I still have to write!

File: index.Rmd (hunk 1 above)
Comment (KevCaz) [reply index.Rmd:136]: All good.

**Pull request merged**